src/main/java/edu/uw/edm/edmzuulproxy/filter/CertificateAuthenticationFilter.java

```diff
@@ -81,6 +88,14 @@ public Object run() {
             final String profilesHeaderValue = profiles != null ? String.join(PROFILES_SEPARATOR, profiles) : "";
             ctx.addZuulRequestHeader(AUTHORIZED_PROFILES_HEADER, profilesHeaderValue);
 
+            if (   certificateToApiKeyMap != null && certificateToApiKeyMap.containsKey(certificateName)
```

Comment:
Consider removing extra spaces in if expression.

Reply:
The extra spaces are for "&&" on the next line so that the conditional expressions are easier to read, I think.